sklearn/metrics/cluster/supervised.py

```diff
@@ -544,6 +589,11 @@ def mutual_info_score(labels_true, labels_pred, contingency=None):
         If value is ``None``, it will be computed, otherwise the given value is
         used, with ``labels_true`` and ``labels_pred`` ignored.
 
+    max_n_classes : int
```

Comment:
You should also (here and in all the docstrings) add a descriptor of the argment; "int, optional (default=5000)"